src/Emulator/Extensions/Mocks/HPSHostController.cs

```diff
@@ -268,5 +372,12 @@ private enum CommonSystemCommands
             Reset = 1,
             Launch = 2
         }
+
+        private enum I2CMessageType
```

Comment:
It's not really I2C Messages, it's our own Socket protocol. Maybe `SocketMessageType` would be a better name?

Reply:
Done